proxy/src/main/resources/rmq.proxy.logback.xml

```diff
@@ -229,10 +229,10 @@
 
     <appender name="RocketmqRebalanceLockAppender_inner"
               class="ch.qos.logback.core.rolling.RollingFileAppender">
-        <file>${user.home}/logs/rocketmqlogs/${brokerLogDir}/lock.log</file>
+        <file>${user.home}${file.separator}logs${file.separator}rocketmqlogs${file.separator}${brokerLogDir:-${file.separator}}${file.separator}lock.log</file>
```

Comment:
If `brokerLogDir` is not set, there will be three consecutive `${file.separator}` here. Have you tested it on multiple operating systems to ensure that the path with three consecutive `${file.separator}` works properly?

Reply:
there will only be one file.separator in real log path

Reply:
@superhx Can you provide test results on MacOS/Windows/Linux, or provide evidence about how it works?

Reply:
Use launch cmd in RocketMQ Quick Start
nohup sh bin/mqnamesrv &
nohup sh bin/mqbroker -n localhost:9876 --enable-proxy &

In my local machine Mac:
![image](https://user-images.githubusercontent.com/5719209/232413538-2e806492-490e-4ff1-9857-82d08d1ba23b.png)

In Linux:
![image](https://user-images.githubusercontent.com/5719209/232419186-946bedec-ca2b-4695-a120-9d1b9547acad.png)


Don't have Windows machine


PS. the proxy logback config path format is the same as rmq.broker.logback.xml